wolfcrypt/src/asn.c

```diff
@@ -11610,22 +11802,24 @@ int GetName(DecodedCert* cert, int nameType, int maxIdx)
     return GetCertName(cert, full, hash, nameType, cert->source, &cert->srcIdx,
                        cert->srcIdx + length);
 #else
-    ASNGetData dataASN[certNameASN_Length];
+    DECL_ASNGETDATA(dataASN, certNameASN_Length);
```

Comment:
Didn't bother with this one as it is too small.
Rule is that we don't want more than 100 bytes of data on stack in small stack case.

Reply:
Reverted back to the way it was.